### PR TITLE
Feature/72 sqlalchemy type support

### LINKED ISSUE #72: sqlalchemy : type support
support more types includes:
- array
- tuple
- map
- uuid

### PR DESCRIPTION
**Pull request by gangtao**
better type support for `array`,`tuple`,`map`

### CODE REVIEW

File: examples/helloworld/dbapi.py [hunk 1]
```diff
@@ -23,3 +22,7 @@
 # in case run following code, it will not stop due to streaming query
 # for row in cusor:
 #     pprint(row)
+
+cusor = conn.execute("select 1=2")
```
Comment (jovezhong) [inline examples/helloworld/dbapi.py]: what does `select 1=2` mean? It's not so common

Comment (gangtao) [reply examples/helloworld/dbapi.py]: it is logical expression , return `False`, check the `bool` type

Comment (jovezhong) [reply examples/helloworld/dbapi.py]: or maybe you want to create a bool, then 
`select 1<>2` will be better

**Review by jovezhong — APPROVED** [review]
no outstanding issue, but I do suggest changing `1=2` to `1<>2`

**Comment by gangtao** [comment]
> no outstanding issue, but I do suggest changing `1=2` to `1<>2`

I moved that sample query  to types

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**